src/driver.ts

```diff
@@ -640,9 +641,14 @@ export abstract class CMakeDriver implements vscode.Disposable {
       else
         return [];
     })();
-    const args = [ '--build', this.binaryDir, '--config', this.currentBuildType, '--target', target, '--' ].concat(
-        generator_args);
-    const child = this.executeCommand(config.cmakePath, args, consumer);
+
+    const args = [ '--build', this.binaryDir, '--config', this.currentBuildType, '--target', target, '--' ]
+                     .concat(generator_args, config.buildToolArgs, config.buildArgs);
```

Comment:
IIRC, `config.buildArgs` goes before the `--` option? I might be wrong, I never used the option myself.

Reply:
I didn't know what the `--` was for, so I let it just be there, but I can swap them around. At least the variable expansion worked like it was supposed to. Like in my other PR, I am waiting for driver testing before merging this one.